src/Jackett.Common/Definitions/ilcorsaronero.yml

```
@@ -7,12 +7,13 @@ type: public
 encoding: Windows-1252
 followredirect: true
 links:
+  - https://ilcorsaronero.in/
```

this redirects to the www.* domain
```suggestion
  - https://www.ilcorsaronero.in/
```